SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/StaticSubobjectsTest/StaticSubobjectTestActor.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "StaticSubobjectTestActor.h"
+#include "Net/UnrealNetwork.h"
+
+AStaticSubobjectTestActor::AStaticSubobjectTestActor()
+{
+	TestIntProperty = 0;
+	bNetLoadOnClient = true;
+	bNetLoadOnNonAuthServer = true;
+	bReplicates = true;
+
+	SetActorLocation(FVector(-20000.0f, -20000.0f, 40.0f));
```

Comment:
This shouldn't be here, change the transform in the spatialNetworkingMap instead.